.github/workflows/ng-renovate.yml

```diff
@@ -16,13 +16,14 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@ac593985615ec2ede58e132d2e21d2b1cbd6127c # tag=v3.3.0
+      # Note: we use `github.run_id` as part of the key hash so that we create a new cache after each run.
       - uses: actions/cache@v3
         with:
           path: |
             .github/ng-renovate/.yarn/cache
-          key: v4-${{hashFiles('.github/ng-renovate/yarn.lock')}}
-          restore-keys: v4-
-
+            /tmp/renovate
+          key: v2-renovate-${{hashFiles('.github/ng-renovate/yarn.lock')}}-${{ github.run_id }}
```

Comment:
Maybe a daily cache would suffice?

Reply:
yeah, I feel like that might be simpler.

Reply:
Updated to use date